src/koa-esm-specifier-transform.ts

```diff
@@ -0,0 +1,57 @@
+import getStream from 'get-stream';
```

Comment:
I think we're still doing copyright notice in all files, yeah?

Reply:
Added to all files.